docs/serialization.md

```diff
@@ -106,6 +106,31 @@ and support all older schemas. This is hoped to be unlikely/rare, but this also
 serves as a check that this is in fact a uhi serialization object. Non-breaking
 changes like additions are allowed without bumping the schema.
 
+## Sparse storage
+
+For sparse histograms, storage contains an `index` key. This is a 2D array; the
+first dimension has the same number of entries as the number of axes, and the
+second dimension has the same number of entries as the number of filled bins.
+There should not be any duplicate entries. The values start at 0 for the
+underflow bin (if there is one), and are in the same order as the axes. The
+data in this case are 1D arrays, one for each bin. For example, if just the `0,
+1` bin was filled with a `5`, then this would be a valid sparse histogram:
+
+```json
+{
+    "storage": {
+        "index": [[0],[1]],
+        "values": [5],
+    }
+}
```

Comment:
I don't get this example. Can you provide a less trivial example where at least two bins are filled?

Specifically, it is not clear to me how the position in the values array maps to the position in the index array.

Reply:
Re-reading it, I think I get it, but adding a second and third value would make this easier to understand I believe. Using one or three values helps to clarify what are axis dimensions and what are value dimensions.

Reply:
I originally had two, but dropped it to one to avoid confusing it with the two axes. I agree three is much clearer, I've changed it to three and added some extra description.